resources/dicts/events/leader_den/success/other_clan.json

```
@@ -1571,6 +1934,17 @@
     "player_clan_temper": ["any"],
     "other_clan_temper": ["wary"]
   },
+  {
+    "interaction_type": "appease",
+    "event_text": "At the Gathering, o_c_n's leader makes the usual accusations and threats. m_c ignores them and singles out an o_c_n couple with the scent of milk on their pelts, and asks in a snarl if they really want their kits to grow up in wartime? o_c_n's leader backs down, shame-faced",
+    "rel_change": 5,
+    "m_c": {
+      "skill": ["INSIGHTFUL,1", "SENSE,1" ],
```

```suggestion
      "skill": ["INSIGHTFUL,1", "SENSE,1", "KIT,1" ],
```
kit could be a fun addition